src/Microsoft.Health.Fhir.Api/Features/ContentTypes/ContentTypeService.cs

```diff
@@ -55,7 +55,7 @@ public async Task CheckRequestedContentTypeAsync(HttpContext httpContext)
                     throw new UnsupportedMediaTypeException(Resources.UnsupportedFormatParameter);
                 }
 
-                string closestClientMediaType = _outputFormatters.GetClosestClientMediaType(resourceFormat, acceptHeaders.Select(x => x.MediaType.Value));
+                string closestClientMediaType = _outputFormatters.GetClosestClientMediaType(resourceFormat, acceptHeaders?.Select(x => x.MediaType.Value));
```

Comment:
Any chance MediaType is null?

Reply:
No, it's a StringSegment, which is a struct.  The documentation says it defaults to Empty.